src/utils/node/dynamicStories.js

```diff
@@ -0,0 +1,67 @@
+import path from "node:path";
+import { existsSync } from "node:fs";
+import { rename, readdir } from "node:fs/promises";
+
+import { vuelessConfig } from "./vuelessConfig.js";
+import { COMPONENTS, VUELESS_DIR, VUELESS_LOCAL_DIR } from "../../constants.js";
+
+const VUELESS_SRC = path.join(VUELESS_DIR, VUELESS_LOCAL_DIR);
```

Comment:
why you creates `node_modules/vueless/src` path? It not exists in any type of env.

You need to use `VUELESS_DIR` instead of `VUELESS_SRC`